Services/CO.CDP.OrganisationInformation.Persistence/Forms/FormQuestion.cs

```diff
@@ -35,7 +35,8 @@ public enum FormQuestionType
     Date,
     CheckBox,
     Address,
-    MultiLine
+    MultiLine,
+    Url
```

Comment:
Do we also need to update the mapping for the data sharing endpoint?

https://github.com/cabinetoffice/GCGS-Central-Digital-Platform/blob/3ed146f0b21806f47aea81fdbdfff45affbbf779/Services/CO.CDP.DataSharing.WebApi/AutoMapper/DataSharingProfile.cs#L122-L169